gatsby-node.js

```diff
@@ -230,6 +230,37 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     })
   })
 
+  // Create contentVersion v2.0 pages
+  const contentV2Pages = [`eth`, `dapps`, `wallets/index`, `what-is-ethereum`]
+  const contentV2Languages = supportedLanguages.filter(
+    (lang) => getLangContentVersion(lang) === 2.0
```

Comment:
```suggestion
    (lang) => getLangContentVersion(lang) >= 2.0
```
Update to GTE (so it includes English, which is 2.4)